packages/analysis/package.json

```diff
@@ -36,7 +36,8 @@
     "@types/lodash": "^4.14.184",
     "@types/react": "^18.2.15",
     "@types/testing-library__jest-dom": "^5.14.9",
-    "jest": "^27.4.3",
+    "jest": "^28.1.1",
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
here and in all other package.json files, we have to explicitly install `jest-environment-jsdom` due to the jest 28 upgrade: https://jest-archive-august-2023.netlify.app/docs/28.x/upgrading-to-jest28#jsdom for a short note; https://jestjs.io/blog/2021/05/25/jest-27#flipping-defaults for longer context